libs/ledger-live-common/package.json

```diff
@@ -295,7 +295,7 @@
     "mkdirp": "*",
     "react": "^17.0.2",
     "react-dom": "17.0.2",
-    "react-native": "0.71.6",
+    "react-native": "0.72.3",
```

Comment:
live-common depends on react-native? 👀  🤯

Reply:
strange indeed, it could be because of the flags in `libs/ledger-live-common/src/data/icons/reactNative` which depend on `react-native-svg` 😬

Reply:
k, hopefully will be addressed once we fully use crypto icons